src/other/utils.jl

```diff
@@ -1,3 +1,15 @@
+"""
+    AsTable
```

Comment:
Maybe mirror `?All`:
```suggestion
    AsTable(cols)
```
The internal field could also use that name.

Reply:
OK - I will rename the internal field